src/notification/discord.ts

```diff
@@ -0,0 +1,30 @@
+import {Webhook, MessageBuilder} from 'discord-webhook-node';
+import {Config} from '../config';
+import {Logger} from '../logger';
+import {Link} from '../store/model';
+
+const hook = new Webhook(Config.notifications.discord.webHookUrl);
+const notifyGroup = Config.notifications.discord.notifyGroup;
+
+export function sendDiscordMessage(cartUrl: string, link: Link) {
+	(async () => {
+		try {
+			const embed = new MessageBuilder();
+			embed.setTitle('Stock Notification');
+			embed.addField('URL', cartUrl, true);
+			embed.addField('Brand', link.brand, true);
+			embed.addField('Model', link.model, true);
+
+			if (notifyGroup !== '') {
+				embed.addField('Attention', notifyGroup, true);
+			}
+
+			embed.setColor(0x76b900);
```

Comment:
I think you want to use a string here.

Reply:
nope just need to use the right case for linter to be happy. This is the correct useage.